Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -97,26 +156,29 @@ public KeyCommand()
                 Description = "Print raw bytes instead of hexadecimal.  No trailing LF appended."
             )]
             bool bytes = false,
-            [Option(Description = "Export a Web3 Secret Storage Formatted json to path.")]
+            [Option(Description = "Export a Web3 Secret Storage Formatted json")]
             bool json = false,
-            [Option(Description = "Path to export json key.")]
-            string path = ""
+            [Option(Description = "Path to key store to export from.")]
+            string? path = null
         )
         {
+            if (path != null)
+            {
+                KeyStore = new Web3KeyStore(Path.GetFullPath(path));
+            }
+
             PrivateKey key = UnprotectKey(keyId, passphrase);
             byte[] rawKey = publicKey ? key.PublicKey.Format(true) : key.ToByteArray();
+            using Stream stdout = Console.OpenStandardOutput();
             if (bytes)
             {
-                using Stream stdout = Console.OpenStandardOutput();
                 stdout.Write(rawKey, 0, rawKey.Length);
             }
             else if (json)
             {
-                Stream fs = PathHandler(path);
                 var ppk = KeyStore.Get(keyId);
-                ppk.WriteJson(fs, keyId);
-                fs.WriteByte(0x0a);
-                fs.Close();
+                ppk.WriteJson(stdout, keyId);
+                stdout.WriteByte(0x0a);
```

Comment:
Maybe this seems like a break-change. However, it may be okay on the Libplanet side 🤔 I don't remember whether the Libplanet has a duty to guarantee backward compatibility or not well.